Tests/ComposableArchitectureTests/EffectCancellationTests.swift

```diff
@@ -83,7 +83,7 @@ final class EffectCancellationTests: XCTestCase {
         .store(in: &self.cancellables)
     }
 
-    _ = XCTWaiter.wait(for: [self.expectation(description: "")], timeout: 0.1)
+    _ = XCTWaiter.wait(for: [self.expectation(description: "")], timeout: 0.3)
```

Comment:
Was this bump necessary?

Reply:
I'm not sure what the tolerances are for writing timing tests that don't flake, but I do think that this timeout has to exceed the delay on `Just(1)`, otherwise it'd be impossible to write a test that fails even when cancellation works.

Reply:
Oh I gotcha, good catch!